pages/search.md

```diff
@@ -1,5 +0,0 @@
----
-title: Search Results
-layout: page-react.html
-entryname: search
----
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-3973d672492af8e66eee88155797fb72R5.